src/explore-education-statistics-api-docs/source/overview/openapi-documents/index.html.md.erb

```diff
@@ -0,0 +1,26 @@
+---
+title: OpenAPI documents
+last_reviewed_on: 2024-09-18
+review_in: 12 months
+weight: 3
+---
+
+# OpenAPI documents
+
+The explore education statistics (EES) API is fully compliant with the [OpenAPI 3.0 specification](https://swagger.io/specification/v3/).
```

Comment:
Would suggest capitalising the first "e"?

Reply:
This would make intuitive sense, but the service name casing was decided by Cam.

I did argue the point that it's not particularly nice and potentially could be unclear to users, but he was pretty insistent. 

Consequently, the convention is:

- Start of sentence - use 'Explore'
- Mid-sentence - use 'explore'

I think at some point he'll want us to go through the service and change it in across the board as well.